docs/ref/contrib/postgres/search.txt

```diff
@@ -254,13 +254,22 @@ however, if you're searching more than a few hundred records, you're likely to
 run into performance problems. Full text search is a more intensive process
 than comparing the size of an integer, for example.
 
+.. versionadded:: 3.2
+
```

Comment:
We don't keep `versionadded`/`versionchanged` annotations for longer than two releases (see [docs](https://docs.djangoproject.com/en/4.1/internals/howto-release-django/#new-stable-branch-tasks). Please remove it.